### PR TITLE
Fix: Print tool errors only once

### PR DESCRIPTION
**Pull request by RaghavRoy145**
Set `SilenceErrors` to `true` for all commands

Signed-off-by: Raghav Roy <raghavroy145@gmail.com>

### CODE REVIEW

File: cmd/labels.go [hunk 1]
```diff
@@ -50,6 +51,7 @@ var labelsFile string
 
 func init() {
 	labelsCmd.Flags().StringVar(&labelsFile, "output", "labels.csv", "output file path")
+	checkURLsCmd.SilenceErrors = true
```
Comment (MadhavJivrajani) [inline cmd/labels.go]: Copy paste error :)
The command names are different every time.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]

File: cmd/labels.go (hunk 1 above)
Comment (MadhavJivrajani) [reply cmd/labels.go]: This is one instance of easy to catch things if we have good test coverage

Comment (RaghavRoy145) [reply cmd/labels.go]: Oh yes, that's my bad. 
And you're right good test coverage would solve the issue of manually checking if the tool works every time.

**Review by MadhavJivrajani — APPROVED** [review]

**Pull request merged**